go/keymanager/tendermint/tendermint.go

```diff
@@ -27,43 +27,28 @@ const BackendName = tmapi.BackendName
 type tendermintBackend struct {
 	logger *logging.Logger
 
-	service  service.TendermintService
+	service service.TendermintService
+	querier *app.QueryFactory
```

Comment:
Is this the right term? Another weird term is `inquirer`, but it also seems wrong :P

Reply:
Both "inquirer" and "querier" are valid.